test/ng/documentSpec.js

```diff
@@ -27,3 +27,29 @@ describe('$document', function() {
     });
   });
 });
+
+
+describe('$$isDocumentHidden', function() {
+  it('should listen on the visibilitychange event', function() {
+    var doc;
+
+    var spy = spyOn(document, 'addEventListener').and.callThrough();
+
+    inject(function($$isDocumentHidden, $document) {
+      expect(spy.calls.mostRecent().args[0]).toBe('visibilitychange');
+      expect(spy.calls.mostRecent().args[1]).toEqual(jasmine.any(Function));
```

Comment:
Since we don't really care if it was the most recent, you could also usr `toHaveBeenCalledWith()`.